x/accountedpool/spec/README.md

```diff
@@ -0,0 +1,29 @@
+# Elys AccountedPool Module
+
+## Contents
+
+1. **[Concepts](01_concepts.md)**
+2. **[Usage](02_usage.md)**
+3. **[Keeper](03_keeper.md)**
+4. **[Protobuf Definitions](04_protobuf_definitions.md)**
+5. **[Functions](05_functions.md)**
+
+## References
+
+Resources:
+
+- [Elys Network Documentation](https://docs.elys.network)
+- [Cosmos SDK Documentation](https://docs.cosmos.network)
+- [GitHub Repository for Elys Network](https://github.com/elys-network/elys)
+
+## Overview
+
+The `accountedpool` module in the Elys Network is designed to manage and maintain the state of accounted pools within the network. It provides functionalities for creating, updating, and querying accounted pools. This module enhances the network by ensuring accurate accounting of pool balances and integrating with other modules like AMM and Perpetual pools.
```

Comment:
I was wondering how is it different from accounting pool state in amm ? is it possible to add why do we add states in accountedPool ?  is it user specific and done to reduce query computation ? thanks

Reply:
can we not account states in amm and perp modules and query ?

Reply:
this module serves as a bridge between amm and perpetual modules so that they dont depend on each other